sorts.js

```diff
@@ -3,3 +3,137 @@ var group2 = [32, 29, 28, 47, 50, 43, 40, 45, 37, 46, 27, 49, 35, 26, 34, 44, 48
 var group3 = ['sadly', 'roast', 'Loopy', 'burger', 'gavel', 'lure', 'postpone', 'frontal', 'Gavel', 'lavender'];
 
 // Your code goes here! Hooray!
+
+// bubbleSorting the first group: group1
+function bubbleSort(group1)
+{
+    var moved;
+    do {
+        moved = false;
+        for (var i=0; i < group1.length-1; i++) {
+            if (group1[i] > group1[i+1]) {
+                var temp = group1[i];
+                group1[i] = group1[i+1];
+                group1[i+1] = temp;
+                moved = true;
+            }
+        }
+    } while (moved);
+}
+ 
+bubbleSort(group1);
+console.log(group1);
+
+
+// bubbleSorting the second group: group2
+function bubbleSort(group2)
```

Comment:
You don't need to redefine bubbleSort here. You just need to call `bubbleSort(group2);`.